src/com/android/kprofiles/battery/KProfilesModesTileService.java

```diff
@@ -18,10 +18,18 @@ public class KProfilesModesTileService extends TileService {
     private static final String KPROFILES_MODES_KEY = "kprofiles_modes";
     private static final String KPROFILES_MODES_NODE = "/sys/module/kprofiles/parameters/kp_mode";
 
+    private boolean iskProfilesModesSupported = FileUtils.fileExists(KPROFILES_MODES_NODE);
```

Comment:
In addition this is a way too long of a name to be comfortable. Also this could and should be static final